MiniApp/Classes/Extensions/MiniApp+NSError.swift

```diff
@@ -0,0 +1,26 @@
+extension NSError {
+
+    class func serverError(code: Int, message: String) -> NSError {
+        return NSError(
+            domain: "APIClient",
+            code: code,
+            userInfo: [NSLocalizedDescriptionKey: message]
+        )
+    }
+
+    class func unknownServerError(httpResponse: HTTPURLResponse?) -> NSError {
+        let unknownError = NSError.serverError(
```

Comment:
Sorry, I missed this one. This can be also converted into a direct return type. Let's do this in the next PR.